apps/bc-wallet-demo-server/src/index.ts

```diff
@@ -10,9 +10,17 @@ import { corsOptions } from './utils/cors'
 import { tractionApiKeyUpdaterInit, tractionGarbageCollection, tractionRequest } from './utils/tractionHelper'
 
 const baseRoute = process.env.BASE_ROUTE
+console.log(`Starting server with BASE_ROUTE: ${baseRoute}`)
+console.log(
+  `Environment variables loaded: ${JSON.stringify({
+    NODE_PORT: process.env.NODE_PORT,
+    BASE_ROUTE: process.env.BASE_ROUTE,
+    CORS_DISABLED: process.env.CORS_DISABLED,
+  })}`,
+)
 
 const app: Express = createExpressServer({
-  controllers: [__dirname + '/controllers/**/*.ts'],
+  controllers: [__dirname + '/controllers/**/*{.js,.ts}'],
```

Comment:
why we just don't 

```suggestion
 controllers: [
    ConnectionController,
    CredentialController,
    DeeplinkController,
    ProofController,
    RevocationController,
    UseCaseController,
    WalletController,
    WebhookController
  ],
```

Reply:
Next time. I have to cook dinner and go out with the kids